apps/explorer/src/components/orders/StatusLabel/index.tsx

```diff
@@ -10,199 +10,141 @@ import {
   IconDefinition,
   faKey,
   faCircleHalfStroke,
+  faSpinner,
+  faExclamationCircle,
+  faQuestionCircle,
 } from '@fortawesome/free-solid-svg-icons'
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import BigNumber from 'bignumber.js'
-import styled, { DefaultTheme, css, keyframes, FlattenSimpleInterpolation } from 'styled-components/macro'
-import { capitalize, formatPercentage } from 'utils'
-
-import { OrderStatus } from 'api/operator'
-
-type CustomOrderStatus = OrderStatus | 'partially filled'
-type DisplayProps = { status: CustomOrderStatus }
-function canBePartiallyFilled(status: string): status is OrderStatus {
-  return ['open', 'cancelling'].includes(status) // expired, cancelled are excluded, they use a custom status "partially filled"
+import { formatPercentage } from 'utils'
+
+import { Wrapper, Label, StyledFAIcon, GenericStatus as StyledGenericStatus } from './styled'
+
+const ORDER_STATUSES = {
```

Comment:
If possible it's better to use enums with Record<>.
It makes code deterministic and protects us from casual mistakes (typescript won't compile if use forget to add a value from some of enum keys).